packages/rke2-ingress-nginx/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -24,9 +24,9 @@
  {{- fail "Controller container image tag should be 0.27.0 or higher" -}}
  {{- end -}}
  {{- end -}}
-@@ -183,3 +183,15 @@
- {{ toYaml .Values.controller.ingressClassResource.parameters | indent 4}}
-   {{ end }}
+@@ -210,3 +210,15 @@
+       mountPath: {{ toYaml "/modules_mount"}}
+ 
```

Comment:
What's this about? Is this just a change to the upstream chart? Sorry, these patches to patches are hard to review.

Reply:
Its just a change from upstream at https://github.com/kubernetes/ingress-nginx/commit/3474c33e15d809ba401b38891a2ed3c4080b751e. Yeah it really is hard to review this. I made sure that all the changes to the defaults we make were not affected.